plugins.json

```diff
@@ -1,26 +1,27 @@
 [
     {
         "name": "Node Manager",
+        "logo": "https://raw.githubusercontent.com/massalabs/thyra-node-manager-plugin/main/front/public/logo.svg",
         "description": "Install and manage Massa nodes from Thyra",
         "assets": {
             "windows": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_windows-amd64.zip",
-                "checksum": "02ee5479b18149300815af3ec1b0f6ac"
+                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.10/thyra-plugin-node-manager_windows-amd64.zip",
+                "checksum": "877c15e37ee4b9a6880467a0e5233212"
             },
             "linux": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_linux-amd64.zip",
-                "checksum": "4b7ba25d612ea41066a8cc679420a60d"
+                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.10/thyra-plugin-node-manager_linux-amd64.zip",
+                "checksum": "e182d77e6773b8e5745dfaf29e56ee55"
             },
             "macos-arm64": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_darwin-arm64.zip",
-                "checksum": "3feb252f7a57e8a393d5ccfc1604f2ea"
+                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.10/thyra-plugin-node-manager_darwin-arm64.zip",
+                "checksum": "421f7b27ff1bede9997cd39e086567d8"
             },
             "macos-amd64": {
-                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.8/thyra-plugin-node-manager_darwin-amd64.zip",
-                "checksum": "247655ea2985c488c678e195699684d7"
+                "url": "https://github.com/massalabs/thyra-node-manager-plugin/releases/download/v0.0.10/thyra-plugin-node-manager_darwin-amd64.zip",
+                "checksum": "a179319ee6d8e5caae3f945989a8135a"
             }
         },
-        "version": "0.0.8",
+        "version": "0.0.18",
```

Comment:
why not the same version as the release, v0.0.10?

Reply:
this was just for the tests

Reply:
> no blocking, I would have done it differently, I'm just sharing with you: create a function that validate the json store, add unit tests for this function with one invalid json store for every conditions you want to check. in the CI, run this function against the json store and return exit code 1 if not valid. This way, you tests all the code with datasets for all possibilities (all failing cases).

I think that you are right it makes more sense. Do you agree @mazmassa ?